api/src/main/java/org/apache/cloudstack/api/command/user/network/CreateNetworkACLListCmd.java

```diff
@@ -128,7 +128,7 @@ public long getEntityOwnerId() {
         } else {
             account = CallContext.current().getCallingAccount();
             if (!Account.Type.ADMIN.equals(account.getType())) {
-                logger.warn(String.format("Only Root Admin can create global ACLs. Account [%s] cannot create any global ACL.", account));
+                logger.error("Only Root Admin can create global ACLs. {} cannot create any global ACL.", account);
```

Comment:
I left only the account object to be logged because its `toString()` already prints the necessary data; soon I'll post the tests.

Reply:
Example without the changes:

```
2025-12-16 17:26:41,858 WARN  [o.a.c.a.c.u.n.CreateNetworkACLListCmd] (qtp2038105753-22:[ctx-a568fe7b, ctx-db4621bd]) (logid:ef24b1da) Only Root Admin can create global ACLs. Account [Account [{"accountName":"test","id":4,"uuid":"991ecfb6-b09a-4abf-9488-7e3d47b175fe"}]] cannot create any global ACL.
```

---

Example with the changes:

```
2025-12-16 19:04:43,593 ERROR [o.a.c.a.c.u.n.CreateNetworkACLListCmd] (qtp2038105753-27:[ctx-37d7f311, ctx-ddfdecd2]) (logid:62794f63) Only Root Admin can create global ACLs. Account [{"accountName":"test","id":4,"uuid":"991ecfb6-b09a-4abf-9488-7e3d47b175fe"}] cannot create any global ACL.

```